### PR TITLE
build(bazel): update to nodejs rules 0.27.7

### PR DESCRIPTION
**Pull request by gregmagolan**
* adds shard_count to some jasmine_node_test target with a lot of specs (this is now supported by jasmine_node_test)
* cleans up `tools/testing/init_node_spec.ts` jasmine bootstrap with added `@bazel/jasmine` `boot()` function:
```
// Initialize jasmine with @bazel/jasmine boot() function. This will initialize
// global.jasmine so that it can be patched by zone.js jasmine-patch.js.
require('@bazel/jasmine').boot();
import 'zone.js/dist/jasmine-patch.js';
```

### CODE REVIEW

File: tools/testing/init_node_spec.ts [hunk 1]
```diff
@@ -21,7 +21,15 @@ import 'reflect-metadata/Reflect';
 // jasmine_node_test under Bazel will check if `jasmineCore.boot(jasmineCore)`
 // has been called and re-use the env if it has.
 // See https://github.com/bazelbuild/rules_nodejs/pull/539
-const jasmineCore: any = require('jasmine-core');
+let jasmineCore: any = null;
+try {
```
Comment (alexeagle) [inline tools/testing/init_node_spec.ts]: I think we want the pass-through require model here, like

`const jasmineCore: any = require('@bazel/jasmine').jasmineCore`

or maybe a bit of refactoring

`require('@bazel/jasmine').boot(jasmineCore)`

then we don't need to try multiple locations

Comment (gregmagolan) [reply tools/testing/init_node_spec.ts]: I like that

Comment (gregmagolan) [reply tools/testing/init_node_spec.ts]: Done

**Review by alexeagle — APPROVED** [review]

**Review by alexeagle — APPROVED** [review]

**Comment by alexeagle** [comment]
caretaker assist: using my global approval powers

**Pull request closed without merging**